app/controllers/catalog_controller.rb

```diff
@@ -97,7 +97,7 @@ def self.modified_field
     config.add_facet_field 'has_journal_sfacet', limit: 5, label: 'Journal Title'
     config.add_facet_field 'language_label_ssim', label: 'Language', limit: 5
     config.add_facet_field 'license_label_ssim', label: 'License', limit: 5
-    config.add_facet_field solr_name('based_near_label', :facetable), label: 'Location', limit: 5
```

Comment:
Why are we changing this name? Could we just keep the name but change what is stored in the field?